src/vtt-segment-loader.js

```diff
@@ -307,19 +291,14 @@ export default class VTTSegmentLoader extends SegmentLoader {
    * @private
    */
   parseVTTCues_(segmentInfo) {
-    let decoder;
-    let decodeBytesToString = false;
-
-    if (typeof window.TextDecoder === 'function') {
```

Comment:
why not keep this? It'll make it less likely that we'll end up with garbled text.

Reply:
`TextDecoder.decode()` only takes an `ArrayBuffer` so we'd have to convert from text to a buffer back to text. Is this what we want?
https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder/decode

Reply:
Oh, I see. In which case, maybe we should just use the StringDecoder to match what we do in Video.js?

Reply:
Also, I wonder if that means that what's happening in vtt.js is not exactly right?
https://github.com/videojs/vtt.js/blob/1f41da80cbd0bb08543d210fbfdbdb3f3d06ba3b/lib/vtt.js#L1014
https://github.com/videojs/vtt.js/blob/1f41da80cbd0bb08543d210fbfdbdb3f3d06ba3b/lib/vtt.js#L1028-L1037

Reply:
We originally used StringDecoder and that was causing issues

Reply:
it was an issue because the source itself was already garbled and we weren't decoding things properly. Given that non-manifest captions worked, it can't be an issue since that's what we use in Video.js.